### PR TITLE
Feature/styled components

### PR DESCRIPTION
**Pull request by lakooi**
- Changing all css from emotion to styled components

### CODE REVIEW

File: frontend/src/components/parentData/ParentBuild.js [hunk 1]
```diff
@@ -32,9 +32,9 @@ const ParentSeries = () => {
         fetchData();
 
         // returned function will be called on component unmount
-        return () => {
-            dispatch({ type: 'flushParentData' });
-        };
+        //return () => {
```
Comment (turso) [inline frontend/src/components/parentData/ParentBuild.js:35]: why has this been removed?

File: frontend/src/components/parentData/ParentSeries.js [hunk 2]
```diff
@@ -32,9 +32,9 @@ const ParentSeries = () => {
         fetchSeriesData();
 
         // returned function will be called on component unmount
-        return () => {
-            dispatch({ type: 'flushParentData' });
-        };
+        //return () => {
```
Comment (turso) [inline frontend/src/components/parentData/ParentSeries.js:35]: same as the above

File: frontend/src/components/parentData/ParentBuild.js (hunk 1 above)
Comment (lakooi) [reply frontend/src/components/parentData/ParentBuild.js:35]: Originally faced some load time issues and removing the flushing fixed it as weird as it sounds. The load was stuck in a loop which might have been caused by the dispatch method. I will try removing this change.

File: frontend/src/components/dashboard/Series.js [hunk 3]
```diff
@@ -2,15 +2,42 @@
 import SuiteInstability from '../../components/graphs/SuiteInstability';
 import TimeLineChart from '../../components/graphs/TimeLineChart';
 import DashboardList from '../dashlist/ListMain';
+import styled from 'styled-components';
+import { dashboardElement } from '../../styles/baseComponents'
```
Comment (turso) [inline frontend/src/components/dashboard/Series.js]: super minor but missing semicolon. Codacy does not like it :)

**Pull request merged**